### DISCUSSION TITLE
An upcoming theme announcement

### DISCUSSION THREAD

**Discussion by warmachine028** [warmachine028/Better-Calculator#17] [Show and tell]
The primary themes will stay as usual but user can apply this theme by editing ```themes.json```.
The new theme is Midnight City inspired from 2077.
## THE PROTOTYPE OF A NEW UPCOMMING THEME : _MIDNIGHT CITY_
![0004](https://user-images.githubusercontent.com/75939390/105948842-b483e800-6091-11eb-859c-4c4b340f2ed4.jpg)
## Main screen might looks like this
![0005](https://user-images.githubusercontent.com/75939390/105948836-b188f780-6091-11eb-87b3-caf4275626fb.jpg)

Please give your feedbacks upon this.

**Comment by ryanraposo** [comment]
:O :O :O 

THIS IS SO GREAT
